src/meta_schedule/tune_context.cc

```diff
@@ -89,6 +89,8 @@ TVM_REGISTER_GLOBAL("meta_schedule.TuneContext")
     });
 
 TVM_REGISTER_GLOBAL("meta_schedule._SHash2Hex").set_body_typed(SHash2Hex);
+TVM_REGISTER_GLOBAL("meta_schedule.TuneContextNodeInitialize")
+    .set_body_method<TuneContext>(&TuneContextNode::Initialize);
```

Comment:
Following the naming convention, let's remove "Node" and use `TuneContext` directly :-)

```suggestion
TVM_REGISTER_GLOBAL("meta_schedule.TuneContextInitialize")
    .set_body_method<TuneContext>(&TuneContextNode::Initialize);
```